### PR TITLE
Issue #3659: Add a lazy-loading option for image fields.

### LINKED ISSUE #3659: [SEO][WP] Add a lazy-loading option for image fields
**Description of the need**

Native `` lazy-loading is coming to the web! https://bit.ly/loading-attribute … `` defers offscreen images until the user scrolls near them. Shipping in Chrome ~75 https://bit.ly/loading-i2s 

This was one of the recommendations Google PageSpeed gave me for one of my Backdrop sites.

We should support this in Backdrop core!

---

References:
* https://css-tricks.com/native-lazy-loading/
* https://web.dev/native-lazy-loading/

---

Advocate: @klonos 

---

PR that adds a global setting under 'Performance': ~https://github.com/backdrop/backdrop/pull/3256~

PR that adds a setting for image fields: https://github.com/backdrop/backdrop/pull/3273

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/3659

### CODE REVIEW

File: core/modules/image/image.field.inc [hunk 1]
```diff
@@ -607,6 +608,24 @@ function image_field_formatter_settings_form($field, $instance, $view_mode, $for
     ),
   );
 
+  if (!isset($settings['image_load'])) {
+    $settings['image_load'] = 'normal';
```
Comment (indigoxela) [inline core/modules/image/image.field.inc]: I suppose, this should be "auto" instead of "normal".

Comment (jenlampton) [reply core/modules/image/image.field.inc]: good catch, thank you!

**Comment by jenlampton** [comment]
Pushed a new PR, now with tests!

**Comment by quicksketch** [comment]
Merged into 1.x for 1.17.0! Thanks @jenlampton!

**Pull request merged**